tools/fix_auth/auth_config_model.rb

```diff
@@ -41,6 +41,10 @@ def recrypt(old_value, options = {})
 
         symbol_keys ? hash.deep_symbolize_keys! : hash.deep_stringify_keys!
         hash.to_yaml
+      rescue ArgumentError # undefined class/module
```

Comment:
This is not necessary, and it currently still raises an error.
I just put this in here to help us diagnose the future serialization issues.

@Fryguy Would you prefer I remove the debugging down below?